main.go

```diff
@@ -46,23 +46,23 @@ func main() {
 	if err := app.WaitForBuilds(buildSlugs, func(build bitrise.Build) {
 		var failReason string
 		var buildURL = fmt.Sprintf("(https://app.bitrise.io/build/%s)", build.Slug)
-		switch build.Status {
-		case 0:
+
+		if build.IsRunning() {
 			log.Printf("- %s %s %s", build.TriggeredWorkflow, build.StatusText, buildURL)
-		case 1:
+		} else if build.IsSuccessful() {
 			log.Donef("- %s successful %s)", build.TriggeredWorkflow, buildURL)
-		case 2:
+		} else if build.IsFailed() {
 			log.Errorf("- %s failed", build.TriggeredWorkflow)
 			failReason = "failed"
-		case 3:
+		} else if build.IsAborted() {
 			log.Warnf("- %s aborted", build.TriggeredWorkflow)
 			failReason = "aborted"
-		case 4:
```

Comment:
@gaborszakacs This was previously set to recognize a status of `4` as "Cancelled". Did this change to mean "Aborted With Success"?

Reply:
Status 4 has always been "Aborted With Success", it's a bit confusing name (and concept), we might use "Cancelled" on the UI for this, which is similarly confusing IMO..

Reply:
ok, I guess for now I'll just leave the log message as-is then